cachecontrol.go

```diff
@@ -52,16 +52,24 @@ func ParseCacheControl(input string) (CacheControl, error) {
 	return cc, nil
 }
 
-func readString(subject string, offset int, endchars string) string {
-	var accum []rune
-	for _, b := range subject[offset:] {
-		if strings.Index(endchars, string(b)) != -1 {
+func contains(set string, r rune) bool {
+	for _, ch := range set {
+		if r == ch {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Can this be replaced by [`strings.ContainsAny`](https://godoc.org/strings#ContainsAny)?